specification/columns/skupricedetails.md

```diff
@@ -0,0 +1,54 @@
+# SKU Price Details
+
+The **SKU Price Details** column represents a list of relevant properties shared by all resources having the same [**SKU Price ID**](#skupriceid). These properties enable a practitioner to more easily calculate metrics such as total units of a service when it is not directly billed in those units (e.g. cores) and thus enables FinOps capabilities such as unit economics. They also help a practitioner understand the specifics of a **SKU Price ID**.
+
+
+The _SkuPriceDetails_ column adheres to the following requirements:
+
+* The _SkuPriceDetails_ column MUST be in [_KeyValueFormat_](#key-valueformat).
+* The key for a property SHOULD be formatted in [PascalCase](#glossary:pascalcase).
+* The properties (both keys and values) contained in the _SkuPriceDetails_ column MUST be shared across all charges having the same _SkuPriceId_.
+* The key for a property SHOULD remain consistent across comparable SKUs having that property and the values for this key SHOULD remain in a consistent format.
```

Comment:
Do we want to say it MUST be consistent across rows with the same SkuId? We can add that later in a separate PR. Just thinking out loud.

Reply:
This line is communicating that you shouldn't include a property of {"Cores" : 12} in one SkuId/SkuPriceId and a property of {"vCPUs" : "8.0"} in another related one. I feel that going more strongly on that statement would almost necessitate us to define specific properties we desire to be used.

Reading your comment another way, it's already explicit in the definition of SkuPriceId that it's a child of SkuId: "A given value of SkuPriceId MUST be associated with one and only one SkuId, except in cases of commitment discount flexibility."